QSB/Messaging/MessageHandler.cs

```diff
@@ -58,6 +57,32 @@ public void SendToAll(T message)
 			QNetworkServer.SendToAll(_eventType, message);
 		}
 
+		public void SendToHost(T message)
+		{
+			if (!QSBNetworkManager.Instance.IsReady)
+			{
+				return;
+			}
+
+			QNetworkServer.SendToClient(0, _eventType, message);
```

Comment:
we cannot guarantee the host will have id 0

Reply:
actually, we can. SendToHost is only called by the server, and the server's local connection always id 0. see QNetworkServer:163